test/test.make

```diff
@@ -134,7 +134,7 @@ RUNTIME_DEPS += LC_ALL=C LANG=C sort -u
 # (https://github.com/intel/pmem-csi/pull/420#discussion_r346850741).
 .PHONY: run_tests
 test: run_tests
-RUN_TESTS = TEST_WORK=$(abspath _work) \
+RUN_TESTS = TEST_WORK=$(abspath _work) REPO_ROOT=`pwd` \
```

Comment:
Did this break the image file tests? https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/job/pmem-csi/view/change-requests/job/PR-777/1/

Reply:
This was a bit strange to me. I guess `pkg/imagefile/test` is supposed to be run by e2e test, but not skipped from `make test`. It was not an issue before as those tests check if REPO_ROOT and CLUSTER are set and exit without running if those are missing. Whereas now they get to run as both variables are preset, which results in failure.

Now I added them to the skip list along with device manager tests.